### PR TITLE
Add WAMPL token to Base Bridges

### PR DESCRIPTION
**Pull request by brandoniles**
**Description**

Add WAMPL token to the Base bridges

@roberto-bayardo

### CODE REVIEW

**Comment by roberto-bayardo** [comment]
Getting a couple of validation errors:

```
error: WAMPL on chain goerli token 0x3b624861a14979537DE1B88F9565F41a7fc45FBf has incorrect symbol
error: WAMPL on chain base-goerli token 0xeAeAdAC73baaF4cB8B024dE9D65B2eeFa722856C does not match L1 token address
```

You can create name overrides in the data.json to address these mismatches.

```
"overrides": {
        "name": "MyTokenNameOnThisChain"
}
```

**Comment by roberto-bayardo** [comment]
I think the specific issue is the token name is all lowercase on goerli ("wampl") vs WAMPL everywhere else.

**Comment by brandoniles** [comment]
Added an override for the goerli symbol, thanks for the pointer.

I'm not sure what `error: WAMPL on chain base-goerli token 0xeAeAdAC73baaF4cB8B024dE9D65B2eeFa722856C does not match L1 token address` means, though. Shouldn't the addresses be different on different chains?

File: data/WAMPL/data.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "decimals": 18,
+  "description": "Wrapped Ampleforth",
+  "website": "https://ampleforth.org",
+  "twitter": "@ampleforthorg",
+  "tokens": {
+    "ethereum": {
+      "address": "0xEDB171C18cE90B633DB442f2A6F72874093b49Ef"
+    },
+    "goerli": {
+      "address": "0x3b624861a14979537DE1B88F9565F41a7fc45FBf"
```
Comment (roberto-bayardo) [inline data/WAMPL/data.json:13]:
```suggestion
      "address": "0x3b624861a14979537DE1B88F9565F41a7fc45FBf",
```

**Comment by brandoniles** [comment]
Ah, yeah that makes sense. I deployed a new token on base-goerli with the fixed reference 👍

**Review by roberto-bayardo — APPROVED** [review]
good to go from Base side.

**Review by hamdiallam — APPROVED** [review]

**Pull request merged**